test/org/sagebionetworks/bridge/services/SurveyServiceMockTest.java

```diff
@@ -228,9 +226,8 @@ public void logicallyDeleteSurveyNotEmptySharedModules() {
     @Test(expected = BadRequestException.class)
     public void physicallyDeleteSurveyNotEmptySharedModules() {
         when(mockSharedModuleMetadataService.queryAllMetadata(anyBoolean(), anyBoolean(), anyString(), any(),
-                anySetOf(String.class), anyBoolean())).thenReturn(ImmutableList.of(makeValidMetadata()));
+                any(), anyBoolean())).thenReturn(ImmutableList.of(makeValidMetadata()));
 
-        doReturn(ImmutableList.of()).when(mockSchedulePlanService).getSchedulePlans(ClientInfo.UNKNOWN_CLIENT, TEST_STUDY, false);
```

Comment:
Should we fix this for all the other tests as well?

* physicallyDeleteSurveyNotEmptySharedModules()
* deleteSurveyPermanentlyNotConstrainedByScheduleWithMultiplePublishedSurveys()
* deleteSurveyPermanentlyNotConstrainedByCompoundScheduleWithMultiplePublishedSurveys()

Reply:
Added to these tests along with mock of getSurveyAllVersions() which is required when the mocks return values correctly.